aggregator/options.go

```diff
@@ -116,10 +125,10 @@ type options struct {
 	quantilesPool             pool.FloatsPool
 
 	// Derived options.
-	fullCounterPrefix     []byte
-	fullTimerPrefix       []byte
-	fullGaugePrefix       []byte
-	timerQuantileSuffixes [][]byte
+	fullCounterPrefix []byte
+	fullTimerPrefix   []byte
+	fullGaugePrefix   []byte
+	suffixMap         map[policy.AggregationType][]byte
```

Comment:
Map lookups for suffixes are expensive, I'm concerned about doing map lookups multiple times for each timer aggregation given that the timer flushes is the biggest performance bottlenecks on the agg tier.